test/fbe/test_github_graph.rb

```diff
@@ -91,4 +100,22 @@ def test_gets_total_commits_of_repo
     result = g.total_commits('zerocracy', 'baza', 'master')
     assert(result.positive?)
   end
+
+  def test_get_fake_empty_conversations
+    WebMock.disable_net_connect!
+    global = {}
+    options = Judges::Options.new({ 'testing' => true })
+    graph = Fbe.github_graph(options:, loog: Loog::NULL, global:)
+    result = graph.resolved_conversations(nil, 'baza', 172)
+    assert(result.empty?)
+  end
+
+  def test_get_fake_conversations
+    WebMock.disable_net_connect!
+    global = {}
+    options = Judges::Options.new({ 'testing' => true })
+    graph = Fbe.github_graph(options:, loog: Loog::NULL, global:)
+    result = graph.resolved_conversations('zerocracy', 'baza', 172)
+    assert_equal(1, result.count)
```

Comment:
@Suban05 Why result.count is 1? Maybe be 2 nodes

Reply:
@Yegorov `result` is an array. The items of the array are conversations in the form of a hash.
`resolved conversations = 1`, but messages in the conversations is 2.
We are interested in conversations

Reply:
@Yegorov the fake method is already unpacked data, it's just array with only one conversation

Reply:
@Suban05 ok, i got it, but it looks ambiguous, maybe need divided this method?